UPGRADE-2.0.md

```diff
@@ -208,6 +208,84 @@
 
 - The `Client::getDefaultProcessors` method has been removed.
 
+- The `Client::message` method has been removed.
+
+- The `Client::captureQuery` method has been removed.
+
+- The `Client::captureMessage` method has changed its signature by removing the
```

Comment:
Maybe we should add a reason to why we removed that?
Also: it's still possible to send a generic message to sentry like this?

Reply:
As an example: can we easily attach a stack trace to the message?

Reply:
>it's still possible to send a generic message to sentry like this?

It's possible by using the `Client::capture` method as all other `Client::capture<...>` methods just forward the call to it by passing the right options as payload.

>can we easily attach a stack trace to the message?

you can by passing the `exception` option in the `$payload` argument.